bundles/ch.elexis.core.l10n/src/ch/elexis/core/l10n/messages_fr.properties

```diff
@@ -8499,6 +8503,12 @@ SystemPreferencePage_editor_labelText_1 = Station ID unique
 
 SystemPreferencePage_editor_selfContact_label = Contact principal
 
+Template_not_editable = Impossible de renommer
+
+Template_not_editable_text = Les mod�les syst�me ne peuvent pas �tre renomm�s.
```

Comment:
Bitte auch franz. und ital. die Sonderzeichen in UTF-8 Notation \u.... ändern.

https://www.utf8-chartable.de/